peerstore/peerstore.go

```diff
@@ -120,6 +129,18 @@ type AddrBook interface {
 
 	// PeersWithAddrs returns all of the peer IDs stored in the AddrBook
 	PeersWithAddrs() peer.IDSlice
+
+	// SignedRoutingState returns a signed RoutingState record for the
+	// given peer id, if one exists in the peerstore. The record is
+	// returned as a byte slice containing a serialized SignedEnvelope.
+	// Returns nil if no routing state exists for the peer.
+	SignedRoutingState(p peer.ID) []byte
+
+	// SignedRoutingStates returns signed RoutingState records for each of
+	// the given peer ids, if one exists in the peerstore.
+	// Returns a map of peer ids to serialized SignedEnvelope messages. If
+	// no routing state exists for a peer, their map entry will be nil.
+	SignedRoutingStates(peers ...peer.ID) map[peer.ID][]byte
```

Comment:
I think that's an overkill, why do we need this method? It bloats the interface somewhat.